CommunityFridgeMapApi/dependencies/python/storage.py

```diff
@@ -0,0 +1,85 @@
+import os
+import uuid
+from urllib.parse import urlparse, urlunparse
+import boto3
+import botocore
+from botocore.config import Config
+
+def get_s3_client(env=os.getenv("Environment")):
+    endpoint_url="http://localstack:4566/" if env == "local" else None
+    config = Config(
+        signature_version=botocore.UNSIGNED, # Do not include signatures in s3 presigned-urls.
+    )
+
+    return boto3.client(
+        "s3",
+        config=config,
+        endpoint_url=endpoint_url,
+    )
+
+def translate_s3_url_for_client(url: str, env=os.getenv("Environment")) -> str:
+    """
+    This function translates a S3 url to a client-accessible urls.
+
+    From a lambda function's perspective, local AWS gateway is located at "localstack:4566" (within cfm-network).
+    However, this hostname is not available for the host machine.
+    In order to fetch S3 files from a local browser, we need to use "localhost:4566".
+    """
+    if env == "local":
+        parsed_url = urlparse(url)
+        return urlunparse(parsed_url._replace(netloc="localhost:4566"))
+    return url
+
+
+class Storage:
```

Comment:
Should this be called S3Storage? Or something S3 related?

Reply:
same common on the file name

Reply:
The idea was to let lambda handlers persist files without worrying about the underlying implementation. It made sense when I was trying to save files in filesystem for local dev environments, but now it makes less sense with localstack for dev.

Renamed `Storage` to `S3Service` in 1e10fcb6a8406982905bd8000567a4b19faf433b.